acquire/acquire.py

```diff
@@ -1551,6 +1551,18 @@ def print_volumes_overview(target):
     log.info("")
 
 
+def print_acquire_warning(target: Target) -> None:
+    if target.os != "windows":
+        log.warning("========================================== WARNING ==========================================")
+        log.warning("")
+        log.warning(
+            "The operating system '%s' is not officially supported by acquire, and should be considered experimental.",
```

Comment:
I'd change "officially" to something a bit less fearful (because there are modules for collecting stuff from other OS's, so you could argue there is "official support"). Perhaps just saying `Operating system support for '%s' is experimental. Some artifacts may not yet be included and some features may not work as expected.`.

Maybe also add something along the lines of "Please notify upstream for any missing artifacts or features".